specs-go/config.go

```diff
@@ -495,7 +495,7 @@ type WindowsNetwork struct {
 // WindowsHyperV contains information for configuring a container to run with Hyper-V isolation.
 type WindowsHyperV struct {
 	// UtilityVMPath is an optional path to the image used for the Utility VM.
-	UtilityVMPath string `json:"utilityvmpath,omitempty"`
+	UtilityVMPath string `json:"utilityVMPath,omitempty"`
```

Comment:
Yes, I agree it's a fine line, but IMO, I would say `VM` is a very common abbreviation for 'Virtual Machine' and think it's OK as is.